src/qt/managenamespage.cpp

```diff
@@ -120,6 +137,42 @@ void ManageNamesPage::onCopyValueAction()
     GUIUtil::copyEntryData(ui->tableView, NameTableModel::Value);
 }
 
+void ManageNamesPage::onConfigureNameAction()
+{
+    QModelIndexList indexes = GUIUtil::getEntryData(ui->tableView, NameTableModel::Name);
+
+    if (indexes.isEmpty())
+        return;
+
+    if (indexes.size() != 1)
+        return;
+
+    WalletModel::UnlockContext ctx(walletModel->requestUnlock ());
+    if (!ctx.isValid ())
+        return;
+
+    const QModelIndex &index = indexes.at(0);
+    const QString &name = index.data(Qt::EditRole).toString();
+    const std::string &strName = name.toStdString();
+    const QString &initValue = index.sibling(index.row(), NameTableModel::Value).data(Qt::EditRole).toString();
+
+    ConfigureNameDialog dlg(platformStyle, name, initValue, this);
+    dlg.setModel(walletModel);
+
+    if (dlg.exec() != QDialog::Accepted)
+        return;
+
+    const QString &newValue = dlg.getReturnData();
+    const Optional<QString> &transferToAddress = dlg.getTransferTo();
```

Comment:
Same as above.

Reply:
AFAICT `getReturnData` does actually return a reference, so I've left that one intact, but you are correct that `getTransferTo` doesn't return a reference; fixed that one.

Reply:
Makes sense.